lobsterpy/cli.py

```diff
@@ -367,6 +381,14 @@ def get_parser() -> argparse.ArgumentParser:
         "If not set, plots will consists of summed cohps. (This argument works only"
         "for interactive plots) ",
     )
+    interactive_plotter_group.add_argument(
+        "--orbitalresplot",
+        "--orbital-res-plot",
```

Comment:
```suggestion
        "--orbital-plot",
        "--orbitalplot",
```
maybe something like this?